### PR TITLE
Updating rvs binary path.

### PR DESCRIPTION
**Pull request by renjithravindrankannath**
Updating rvs binary path

### CODE REVIEW

File: var/spack/repos/builtin/packages/rocm-validation-suite/package.py [hunk 1]
```diff
@@ -183,10 +183,14 @@ def patch(self):
             filter_file(
                 "@ROCM_PATH@/rvs", self.spec.prefix.rvs, "rvs/conf/deviceid.sh.in", string=True
             )
-        elif "@5.2.0:" in self.spec:
+        elif "@5.2.0:5.4" in self.spec:
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/rocm-validation-suite/package.py]: Version (range) checks should be using `self.spec.satisfies()`.

See line 204 and the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#testing-spec-constraints.

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/rocm-validation-suite/package.py]: Also, I think the range `"@5.2:5.4" would suffice.

File: var/spack/repos/builtin/packages/rocm-validation-suite/package.py [hunk 2]
```diff
@@ -183,10 +183,14 @@ def patch(self):
...
             filter_file(
                 "@ROCM_PATH@/bin", self.spec.prefix.bin, "rvs/conf/deviceid.sh.in", string=True
             )
+        elif "@5.5:" in self.spec:
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/rocm-validation-suite/package.py]: Same here for use of `self.spec.satisfies()`.

File: var/spack/repos/builtin/packages/rocm-validation-suite/package.py (hunk 1 above)
Comment (tldahlgren) [reply var/spack/repos/builtin/packages/rocm-validation-suite/package.py]: I also suggest fixing the check at line 182 too.

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Suggest use of `self.spec.satisfies()` for version range checks.

File: var/spack/repos/builtin/packages/rocm-validation-suite/package.py (hunk 1 above)
Comment (renjithravindrankannath) [reply var/spack/repos/builtin/packages/rocm-validation-suite/package.py]: Updated as recommended.

File: var/spack/repos/builtin/packages/rocm-validation-suite/package.py (hunk 2 above)
Comment (renjithravindrankannath) [reply var/spack/repos/builtin/packages/rocm-validation-suite/package.py]: Updated as recommended

**Review by tldahlgren — APPROVED** [review]
LGTM

**Pull request merged**